specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -248,6 +259,44 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "name": "semanticErrorHandling",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "fail",
+              "partial"
+            ],
+            "x-ms-enum": {
+              "name": "SemanticErrorHandling",
+              "modelAsString": false,
+              "values": [
+                {
+                  "value": "fail",
+                  "name": "Fail",
+                  "description": "Default behavior. If there is an exception during semantic processing, the query will fail and return the appropriate HTTP code depending on the error."
+                },
+                {
+                  "value": "partial",
+                  "name": "Partial",
+                  "description": "If there is an exception during semantic processing, still return partial results. Partial result content depends on what semantic step failed and the reason for the failure."
```

Comment:
I believe the http response code would be 206 if it is partial success.

Reply:
Is that something we want to document here?